core/simulations/scripts/causeDebt.ts

```diff
@@ -0,0 +1,11 @@
+import hre from 'hardhat';
+import { causeDebt } from '../../helpers/causeDebt';
+import { HARDHAT_PRIVATE_KEY } from '../../helpers/constants';
+import { createWalletForRpc } from '../../helpers/hardhat';
+
+const provider = hre.network.provider;
+
+export default async () => {
+    await createWalletForRpc(HARDHAT_PRIVATE_KEY, 'custom');
+    await causeDebt('custom', provider);
+};
```

Comment:
Why do you need to call `createWalletForRpc` twice? Inside `createWalletForRpc` and then also here?

Reply:
i don't understand this question. i think there's a misnaming in your sentence. 

Could you link places where this is called?

Reply:
You first call it here:
https://github.com/sidestream-tech/unified-auctions-ui/blob/21ef5cd07a6cbc3b44a60063a51931ad80f457ee/core/simulations/config.ts#L12 inside `resetFork` "script"

Then you call it in the next step:
https://github.com/sidestream-tech/unified-auctions-ui/blob/21ef5cd07a6cbc3b44a60063a51931ad80f457ee/core/simulations/config.ts#L16 inside `causeDebt` "script"